packages/flutter/lib/src/widgets/router.dart

```diff
@@ -852,10 +853,13 @@ abstract class BackButtonDispatcher extends _CallbackHookProvider<Future<bool>>
   ///
   /// The [BackButtonDispatcher] must have a listener registered before it can
   /// be told to take priority.
-  void takePriority() {
-    if (_children != null)
-      _children!.clear();
-  }
+  ///
+  /// The this method only takes the priority from its parent by default.
+  /// Calling this method alone does not claim the priority to handle Android
+  /// back button event if the parent themself does not have the priority.
+  /// Sets the `applyToParent` parameter to true to ask parent to recursively
+  /// claim priority from their parent.
```

Comment:
What do you mean by this?

Reply:
I thought the idea behind taking priority was that you get the back button events? What does it now mean to call `takePriority `  without setting applyToParent?